applications/StructuralMechanicsApplication/python_scripts/RVEAnalysis.py

```diff
@@ -0,0 +1,353 @@
+# makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication
+import math
+from KratosMultiphysics.StructuralMechanicsApplication.structural_mechanics_analysis import StructuralMechanicsAnalysis
+
+
+class RVEAnalysis(StructuralMechanicsAnalysis):
```

Comment:
```suggestion
class RveAnalysis(StructuralMechanicsAnalysis):
```

Reply:
I would prefer keeping the capitals here, as it is 

Represenative Volume Element

Reply:
ok, no problem